ktest.kts

```diff
@@ -77,7 +89,7 @@ fun checkOrExit(condition: Boolean, msg: () -> String) {
 
 fun String.execute(
     workingDir: File = File("."),
-    timeoutAmount: Long = 60,
+    timeoutAmount: Long = 120,
```

Comment:
timeout was causing some failures when it was taking longer to build, increased it two minutes and seemed to make that issue go away